packages/core/src/fhir-deduplication/resources/family-member-history.ts

```diff
@@ -29,15 +29,11 @@ export function groupSameFamilyMemberHistories(famMemberHists: FamilyMemberHisto
 
   for (const famMemberHist of famMemberHists) {
     const relationship = extractCode(famMemberHist.relationship);
-    const name = famMemberHist.name;
+    const dob = famMemberHist.bornDate;
+    console.log(JSON.stringify({ relationship, dob }));
     // const date = getDateFromResource(famMemberHist, "date"); // We're currently not mapping the date for FamilyMemberHistory.hbs
     if (relationship) {
-      let key;
-      if (relationship === "father" || relationship === "mother") {
-        key = JSON.stringify({ relationship });
-      } else {
-        key = JSON.stringify({ relationship, name });
-      }
+      const key = JSON.stringify({ relationship, dob });
```

Comment:
so your analysis showed no dobs and no names. We definetly need to fix the core mapping on this one, but for now, should we just leave the key as relationship?

Reply:
The DOB mapping exists on the FHIR converter. It just seems that none of the CDAs I tested it with are providing it. So I think we can keep it in the key.